util/config.py

```diff
@@ -7,9 +7,13 @@
 REACT_PORT = 3000
 API_PORT = 8000
 # Data sets
-ROTTEN_TOMATO_PATH = os.path.join('tests','data','rotten_tomatoes')
-RATED_DATASETS_PATH = os.path.join('..','rated_datasets')
+# TODO review this
+# all these paths are relative to the server.py script
+# your os.getcwd() should be "api/"
```

Comment:
Aren't we normally starting the server and importing all the other packages relative to the home of all module folders? E.g. running the server with `python3 api/server.py`